tests/integration/test_adapter.py

```diff
@@ -0,0 +1,145 @@
+from abc import abstractmethod
+
+
+import pytest
+
+
+from openscm.adapter import Adapter
+from openscm.core import ParameterSet
+
+
+@pytest.fixture(scope="function")
+def test_adapter(request):
+    """Get an initialised instance of an the requesting classes ``tadapter`` property.
```

Comment:
We are using different styles here. I prefer the line break after “”” as it looks much tidier and this whitespace won’t hurt in comments. Is there any standard?

Reply:
like so?

```python
"""
Get an initialised instance of an the requesting classes ``tadapter`` property.
"""
```

Happy for us to have that as the standard, I have no preference. black seems to be ambivalent to it so if we want to automate fixing it we might need to get fancier.

Reply:
Cool, would like that ;)